gulpfile.js

```diff
@@ -322,8 +385,9 @@
                 .watch(
                     [
                         '*.js',
-                        'public/app/**',
-                        'test/**/*.js'
+                        'public/app/**/*.js',
```

Comment:
This change means that it doesn't run the tests for template (html file) changes.